src/components/metadata/form/index.js

```diff
@@ -160,7 +164,9 @@ const MetadataFormListing = (props) => {
 
     const saveDisabled = loadingOrSubmitting || !dirty || errors.error;
 
-    const showSaveToFile = !loadingOrSubmitting && !(dirty && editable);
+    const showSaveToFile = !loadingOrSubmitting && !dirty;
```

Comment:
Looks like `editable` got dropped from this boolean - was that intentional?

Reply:
Yes. So that users can save the metadata even if they only have read-only access to the files (eg: community data)